pallets/asset-manager/Cargo.toml

```diff
@@ -37,14 +38,18 @@ std = [
 	"sp-std/std",
 	"manta-primitives/std",
 	"frame-benchmarking/std",
+	"xcm/std",
```

Comment:
I think `xcm/std` and `frame-benchmarking/std` can be removed.
We don't need both in std feature.